### DISCUSSION TITLE
Using tauri_plugin_sql in Rust

### DISCUSSION THREAD

**Discussion by charmbyte** [tauri-apps/tauri#11383] [Q&A]
I couldn't find any examples of this, but is there a way to create a connection and execute queries using the tauri_plugin_sql on the Rust side of things? All I've found are JavaScript examples. Thank you!

**Comment by FabianLars** [comment]
The plugin does not have any Rust apis. Use any of the existing rust crates for this instead. If you chose sqlx you can re-use the plugin's connection pools via `app.state::<tauri_plugin_sql::DbInstances>()`
